x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/actions/duplicate_rule.test.ts

```diff
@@ -248,71 +172,87 @@ describe('duplicateRule', () => {
         expect.objectContaining({
           params: expect.objectContaining({
             immutable: false,
+            ruleSource: {
+              type: 'internal',
+            },
           }),
         })
       );
     });
 
-    it('copies related integrations as is', async () => {
-      const rule = createCustomRule();
-      rule.params.relatedIntegrations = [
-        {
-          package: 'aws',
-          version: '~1.2.3',
-          integration: 'route53',
-        },
-      ];
-
+    it('copies fields from the original rule', async () => {
+      const rule = createPrebuiltRule();
       const result = await duplicateRule({
         rule,
       });
 
-      expect(result).toEqual(
-        expect.objectContaining({
-          params: expect.objectContaining({
-            relatedIntegrations: rule.params.relatedIntegrations,
-          }),
-        })
-      );
+      expect(result).toEqual({
+        name: expect.anything(), // covered in a separate test
+        params: {
+          ...rule.params,
+          ruleId: expect.anything(), // covered in a separate test
+          immutable: expect.anything(), // covered in a separate test
+          ruleSource: expect.anything(), // covered in a separate test
+        },
+        tags: rule.tags,
+        alertTypeId: rule.alertTypeId,
+        consumer: rule.consumer,
+        schedule: rule.schedule,
+        actions: rule.actions,
+        systemActions: rule.actions,
+        enabled: false, // covered in a separate test
+      });
     });
+  });
 
-    it('copies required fields as is', async () => {
-      const rule = createCustomRule();
-      rule.params.requiredFields = [
-        {
-          name: 'event.action',
-          type: 'keyword',
-          ecs: true,
-        },
-      ];
+  describe('when duplicating a custom rule', () => {
+    const createCustomRule = () => {
+      const rule = createTestRule();
+      rule.params.immutable = false;
+      rule.params.ruleSource = {
+        type: 'internal',
+      };
+      return rule;
+    };
 
+    it('keeps it custom', async () => {
+      const rule = createCustomRule();
       const result = await duplicateRule({
         rule,
       });
 
       expect(result).toEqual(
```

Comment:
nit: It should be simpler to assert `result.params` directly

```suggestion
      expect(result.params).toEqual(
        expect.objectContaining({
            requiredFields: rule.params.requiredFields,
            immutable: false,
            ruleSource: {
              type: 'internal',
            },
          }),
        })
     );
```